docs/ci-services.md

```diff
@@ -0,0 +1,55 @@
+# CI Services
+
+The CI service blob defines the supported CI services the user will be
+presented with when running the scaffolder.
+
+## Example
+
+```javascript
+{
+  TravisCI: {scaffolder: travisScaffolder, public: true}
+}
+```
+
+## Config
+
+### keys: __string__
+
+Keys in the CI services blob define the options the user will be presented with
+when choosing their desired CI service.
+
+example: `TravisCI`
+
+### values: __object__
+
+Values in the CI services blob define the actual mechanisms and configuration
+that will be used by the JavaScript Scaffolder to scaffold the CI service for
+the user's project.
+
+#### scaffolder: __function__ (_required_)
+
+Consumer-defined plugin(??) called when this ci service is selected by the
+user in the CLI. Will be called with the following arguments:
+
+* `projectRoot` __string__ full path to directory where scaffolder was initiated
```

Comment:
```suggestion
* `projectRoot` __string__ full path to directory where scaffolder will create the JavaScript project
```

especially now that `add-package-to-monorepo` exists, it is important to clarify that this is not the same as `process.cwd()`. this was always the intent, but that is finally a concrete example where it can be different